### DISCUSSION TITLE
Swap options in shops

### DISCUSSION THREAD

**Discussion by Puzzle-Drops** [runelite/runelite#15199] [Ideas]
![image](https://user-images.githubusercontent.com/86485071/175209148-3f8ea304-2486-4fc0-abd1-31b55d12145d.png)
Include a swap option in shops. I can Swap the menu entry for Bert to just left click Sand. But I can't swap the option from Value to Buy-50 at the shops.

**Comment by Enriath** [comment]
It is already possible to swap the shift click amount for shops. Originally this was a safety measure, so you'd have to explicitly hold shift to click to buy, and some shops have some very expensive items at a relatively high stock.
